SU-SWS/stanford_paragraph_types
# EARTH-000 change module names to keep consistent

# READY FOR REVIEW

# Summary
- Changed `stanford_link_banner` to `stanford_paragraph_link_banner` to keep consistent with the other module names.
- Same for `stanford_link_item`
- Changed file paths to all be `paragraph/[paragraph-type]`

# Needed By (Date)
- August-ish

# Urgency
- low

# Steps to Test

1. Checkout branch here and [BLT](https://github.com/stanford-earth/SE3_BLT/pull/31)
2. config import
3. ensure the module is enabled and paragraph exists.

# Associated Issues and/or People
- https://github.com/stanford-earth/SE3_BLT/pull/31

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
